backend/app/db/models/path.py

```diff
@@ -0,0 +1,11 @@
+from app.db.database import Base
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+
+
+class Path(Base):
+    __tablename__ = "paths"
+
+    id: Mapped[int] = mapped_column(primary_key=True)
```

Comment:
Consider adding a mixing that could have the `id` and `created_at`, you can use it in the models instead of repeating the same line. Such mixing can also include auto generation of the `__table__`.  This can be applied to every model here. [More info](https://docs.sqlalchemy.org/en/20/orm/declarative_mixins.html)